packages/__docs__/src/Theme/index.tsx

```diff
@@ -181,16 +179,24 @@ class Theme extends Component<ThemeProps> {
     const subSections = []
     let baseColors = {}
     const newData = Object.assign({}, data)
-
-    if (name === 'colors' && data.values) {
-      baseColors = data.values
+    if (name === 'colors' && data.primitives) {
+      baseColors = data.primitives
       delete newData.values
 
       this._colorMap = this.mapColors(baseColors)
       subSections.push(<ThemeColors colors={baseColors} />)
     }
 
     Object.keys(newData).forEach((key) => {
+      //primitives are the color palette above
+      if (key === 'primitives') {
+        return
+      }
+
+      // if (key === 'contrasts') {
```

Comment:
why is this commented out? maybe delete it if we won't need it now or add a TODO comment if it's temporary